yb-voyager/cmd/common.go

```diff
@@ -341,6 +341,14 @@ func initMetaDB() {
 	if err != nil {
 		utils.ErrExit("could not init migration status record: %w", err)
 	}
+	msr, err := metaDB.GetMigrationStatusRecord()
+	if err != nil {
+		utils.ErrExit("get migration status record: %v", err)
+	}
+	if msr.VoyagerVersion != utils.YB_VOYAGER_VERSION {
+		utils.ErrExit("It is recommended to use a single yb-voyager version for each export-dir to prevent any issues.\n" +
+		"Please maintain the same version for this export-dir, or use a different export-dir if you need to change the version.")
```

Comment:
`"Voyager requires the entire migration workflow to be executed using a single Voyager version. The export-dir %q was created using version %q and the current version is %q. Either use Voyager %q to continue the migration or start afresh with a new export-dir."`